src/v2/es-utils.js

```diff
@@ -54,6 +54,37 @@ class ESClass {
       });
   }
 
+  async healthy(timeout, status) {
+    const resp = await this.client.cluster.health({
+      level: 'cluster',
+      waitForStatus: status || 'green',
+      timeout: timeout || '90s'
+    });
+    if (resp.hasOwnProperty('body') && resp.body.status !== 'red') {
+      return (resp.body);
+    }
+    const info = await this.client.info();
+    return Object.assign(resp, info);
+  }
+
+  async waitPort() {
+    const opts = {
+      host: this.env.ES_HOST,
+      port: this.env.ES_PORT,
+      output: 'silent',
+      timeout: 120000
+    };
+    await waitOn(opts)
```

Comment:
Is there a more meaningful name than `waitOn`? What is it waiting on?

Reply:
I missed the dependency on `wait-port` and understand this code now - I think I'd apply this naming feedback to `waitPort`.

Reply:
These methods are inside the ESClass which handles all interaction with elasticsearch and as such, the methods refer to the specific function of elasticsearch ... is that acceptable? I added a comment to inform the reader about the purpose of the ESClass ...